firehydrant/fh-docs
# Feature/algolia

- Removed fuse.js and implemented Algolia search with their instantsearch.js module
- Added required styles + overrides
- Modified placeholder containers and HTML for Algolia to dynamically insert search results

**How to Test**:

- Pull/checkout this branch
- `npm i`
- `npm start`
- Type some stuff into the search bar. Verify you're seeing results. Should see a maximum of 3 paragraphs per page if there are matches
- Verify readability of search results + highlighted matches on dark and light modes
- While cursor focused on the search bar, hitting `Esc` should blur(), delete search bar contents, and hide the search results
